src/parser/jobs/rpr/modules/Gauge.tsx

```diff
@@ -0,0 +1,178 @@
+import {Trans} from '@lingui/react'
+import Color from 'color'
+import {DataLink} from 'components/ui/DbLink'
+import {ActionKey} from 'data/ACTIONS'
+import {Event} from 'event'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {CounterGauge, Gauge as CoreGauge} from 'parser/core/modules/Gauge'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+const SOUL_MAX_VALUE = 100
+const SHROUD_MAX_VALUE = 100
```

Comment:
Prefer to keep all the soul stuff together, then have a separate block for shroud, and slap a comment above them to indicate as such.